microsoft-dynamics-365/2.5/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,97 @@
+= Microsoft Dynamics 365 Connector 2.4 - Mule 4
+:page-aliases: connectors::microsoft/microsoft-dynamics-365-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Microsoft Dynamics 365 (Microsoft Dynamics 365 Connector) enables integration with the Microsoft Dynamics 365 Cloud API.
+
+This connector enables you to perform operations to:
+
+* Authorize or unauthorize server access
+* Create, update, and delete entities
+* Retrieve a single entity or query multiple entities
+* Associate and disassociate entities
+* Execute actions
+
+Release Notes: xref:release-notes::connector/microsoft-dynamics-365-connector-release-notes-mule-4.adoc[Microsoft Dynamics 365 Connector Release Notes] +
+Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-microsoft-dynamics365-connector/[Microsoft Dynamics 365 Connector]
+
+== Prerequisites
+
+Before creating an app, you must have access to the Microsoft Dynamics 365 target resource and
+Anypoint Platform. You also must understand how to create a Mule app using
+Design Center or Anypoint Studio.
+
+Obtain a client ID and secret for your app by logging in to the Microsoft Azure Active Directory portal at portal.azure.com.
+
+Before starting:
+
+You need access to a Microsoft Dynamics 365 instance for which Azure Active Directory is the Identity Provider that provides access to an application.
+
+== Common Use Cases
+
+The following use cases are described in the xref:microsoft-dynamics-365-connector-examples.adoc[Examples] topic.
+
+* Create an account with attributes that associate an account with a contact
+* Create an empty contact entity
+* Create an empty opportunity entity
+* Create multiple entities of the same type in a single batch request
+* Delete an entity by type
+* Disassociate keys based on an identity ID
+* Retrieve an entity by type
+* Retrieve multiple entities using a URL request
+* Retrieve multiple entities using DataSense query language
+* Update an account with attributes
+* Update multiple entities of the same type in a single batch request
+
+== Audience
+
+* New users, read:
+** xref:microsoft-dynamics-365-connector-studio.adoc[Use Studio to Configure the Connector] to create your Mule app
+** xref:microsoft-dynamics-365-connector-examples.adoc[Examples], which provides an example use case for the connector
+* Advanced users, read:
+** xref:microsoft-dynamics-365-connector-xml-maven.adoc[XML and Maven Support]
+** xref:microsoft-dynamics-365-connector-examples.adoc[Examples], which provides an example use case for the connector
+
+== Authentication Types
+
+Microsoft Dynamics 365 Connector uses the following authentication types:
+
+* *Connection* (OAuth 2.0) +
+Connects to servers that manage authorization and token access
+* *OAuth Username and Password* +
+Uses a username and password to authenticate the connection to the server
```

Comment:
In this release, we will add support for OAuth Client Credentials as well; I think it should be added here as well?